css-contain-2/Overview.bs

```diff
@@ -544,40 +597,32 @@ Size Containment</h3>
 		contain-size-replaced-006.html
 		</wpt>
 
-	2. [=Size containment boxes=] must be treated
-		as if they had no [=intrinsic aspect ratio=].
-
-		Note: Size containment only suppresses the [=intrinsic aspect ratio=],
-		not [=preferred aspect ratio=],
-		so properties like 'aspect-ratio' which affect that [=preferred aspect ratio=] directly
-		are honored.
-
-		<div class=example>
-			Given the following markup and style, the image would be sized to 100px by 100px,
-			as the aspect ratio set by the 'aspect-ratio' property takes effect.
-			<pre><code class=lang-css>
-			img {
-				width: 100px;
-				aspect-ratio: 1/1;
-				contain: size;
-			}
-			</code><code class=lang-html>
-			&lt;img src="https://www.example.com/300x100.jpg">
-			</code></pre>
-
-			If the 'aspect-ratio' property had not been declared,
-			the image would have been 100px by 0px,
-			as its intrinsic aspect ratio is suppressed,
-			and its intrinsic height is treated as 0.
-		</div>
-
 	3. [=Size containment boxes=] are <a spec=css-break-3>monolithic</a> (See [[CSS-BREAK-3#possible-breaks]]).
 
 		<wpt>
 		contain-size-breaks-001.html
 		contain-size-monolithic-001.html
 		</wpt>
 
+	<div class=example>
+		Given the following markup and style, the image would be sized to 100px by 100px,
+		as the aspect ratio set by the 'aspect-ratio' property takes effect.
+		<pre><code class=lang-css>
```

Comment:
Just checking - does this highlight properly?

Reply:
Yes. I was hoping it would work, and it just did. Is the markup considered bad?